src/models/offer.js

```diff
@@ -54,21 +84,37 @@ const offerSchema = new Schema(
       type: [
         {
           question: {
-            type: String
+            type: String,
+            required: [true, FIELD_CANNOT_BE_EMPTY('question')],
+            validate: {
+              validator: (question) => {
+                return question.trim().length > 0
+              },
+              message: 'Question cannot contain only whitespace'
+            }
           },
           answer: {
-            type: String
+            type: String,
+            required: [true, FIELD_CANNOT_BE_EMPTY('answer')],
+            validate: {
+              validator: (answer) => {
+                return answer.trim().length > 0
+              },
+              message: 'Answer cannot contain only whitespace'
+            }
```

Comment:
use trim option and minLength validator [look here](https://mongoosejs.com/docs/schematypes.html#string-validators)